src/utils/core.py

```
@@ -0,0 +1,206 @@
+
+            url = f"https://discord.com/api/v10/applications/{app_id}/guilds/{self.target_guild_id}/commands"
+            token = os.getenv('DISCORD_TOKEN')
+            if not token:
+                logger.warning("🔍 DISCORD_TOKEN not available in environment for diagnostic request")
+                return
+
+            headers = {"Authorization": f"Bot {token}"}
+
+            # Use our existing aiohttp session (created in setup_hook) if available
+            session = self.session or aiohttp.ClientSession()
+            close_session = self.session is None
```

Ruff (0.14.1)

Local variable `close_session` is assigned to but never used

Remove assignment to unused variable `close_session`

(F841)